king_phisher/client/application.py

```diff
@@ -101,14 +101,24 @@ class KingPhisherClientApplication(_Gtk_Application):
 	# pylint: disable=too-many-public-methods
 	__gsignals__ = {
 		'campaign-changed': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'campaign-created': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'campaign-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
 		'campaign-set': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'config-load': (GObject.SIGNAL_RUN_LAST, None, ()),
 		'config-save': (GObject.SIGNAL_RUN_LAST, None, ()),
+		'credential-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
 		'exit': (GObject.SIGNAL_RUN_LAST, None, ()),
 		'exit-confirm': (GObject.SIGNAL_RUN_LAST, None, ()),
+		'message-deleted': (GObject.SIGNAL_RUN_FIRST, None, (str,)),
+		'message-sent': (GObject.SIGNAL_RUN_LAST, None, (str, str)),
 		'reload-css-style': (GObject.SIGNAL_RUN_FIRST, None, ()),
 		'rpc-cache-clear': (GObject.SIGNAL_RUN_FIRST, None, ()),
-		'server-connected': (GObject.SIGNAL_RUN_LAST, None, ())
+		'server-connected': (GObject.SIGNAL_RUN_LAST, None, ()),
+		'server-disconnected': (GObject.SIGNAL_RUN_FIRST, None, ()),
+		'sftp-client-start': (GObject.SIGNAL_RUN_FIRST, None, ()),
```

Comment:
Shouldn't this be `GObject.SIGNAL_RUN_LAST` so if a plugin wants to start a custom sftp client, they can do it before the default handler kicks in and mark the signal as handled?